gyronimo/metrics/morphism_helena.hh

```diff
@@ -20,27 +20,36 @@
 #ifndef GYRONIMO_MORPHISM_HELENA
 #define GYRONIMO_MORPHISM_HELENA
 
+#include <gsl/gsl_vector.h>
+#include <gsl/gsl_multiroots.h>
+#include <numbers>
+#include <gyronimo/core/error.hh>
+#include <gyronimo/core/multiroot.hh>
 #include <gyronimo/metrics/morphism.hh>
```

Comment:
Why have all these new includes been added? If they are not explicitly needed in this include file, they should be removed.